src/StorageDirectories.php

```diff
@@ -26,11 +28,15 @@ public static function create()
             self::Path . '/framework/views',
         ];
 
-        foreach ($directories as $directory) {
-            if (! is_dir($directory)) {
-                fwrite(STDERR, "Creating storage directory: {$directory}" . PHP_EOL);
+        $directories = array_filter($directories, static fn ($directory) => ! is_dir($directory));
```

Comment:
I've never seen `static` closures. What's that for?

Reply:
@tillkruss PhpStorm's PHP inspection EA Extended plugin often suggests that refactoring (making the function static) for micro-optimizations (e.g. https://stackoverflow.com/a/64481894/245552). I ignore it 99% of the time because it is noise, but here I figured "why not 🤷". I can't tell if there's a real performance difference, but this code might be called a lot so it doesn't hurt.